BossMod/Autorotation/Utility/RoleMagicalUtility.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace BossMod.Autorotation;
+
+public abstract class RoleMagicalUtility(RotationModuleManager manager, Actor player) : GenericUtility(manager, player)
```

Comment:
Should probably be called 'caster' rather than 'magical', to match Role enum value...

Reply:
Fixed.